controllers/event_rsvp.rb

```diff
@@ -43,14 +43,23 @@ def remove
     end
   end
 
+  def get_users
+    begin
+      users = User.find(rsvp_params[:user_ids])
+      render_json_dump(success_json.merge(users: serialize_data(users, BasicUserSerializer)))
+    rescue
```

Comment:
Rather than rescuing this we should be only adding the user json if there are users. Rescue should be reserved for actual errors, rather than empty results.

We need to do something like:

```
users = User.find(rsvp_params[:user_ids])
result =  success_json

if users.any?
  success_json.merge(users: serialize_data(users, BasicUserSerializer))
end

render_json_dump(result)
```

Reply:
@angusmcleod I considered the case where a wrong user id is passed. As it is a get request, that's possible, and in that case `ActiveRecord::RecordNotFound` error is thrown which I'm rescuing from.

I'll add the case if array is empty which I didn't consider previously.